src/commands/synthetics/__tests__/mobile.test.ts

```diff
@@ -21,412 +21,414 @@ import {
   getMockAppUploadReporter,
 } from './fixtures'
 
-describe('getSizeAndPartsFromFile', () => {
-  test('correctly get size and parts of a file', async () => {
-    const tmpdir = fs.mkdtempSync('getSizeAndPartsFromFile')
-    try {
-      // write test content to a file in the temporary directory
-      const filename = path.join(tmpdir, 'compute_md5_test')
-      const fileContent = '7 bytes'
-      fs.writeFileSync(filename, fileContent)
-
-      expect(await mobile.getSizeAndPartsFromFile(filename)).toEqual({
-        appSize: 7,
-        parts: [
-          {
-            blob: Buffer.from(fileContent),
-            md5: 'QCi9PCxLLuyHmU0aRshoeA==',
-            partNumber: 1,
-          },
-        ],
-      })
-    } finally {
-      // always clean up created tmpdir
-      fs.rmSync(tmpdir, {recursive: true, force: true})
-    }
+describe('mobile tests', () => {
```

Comment:
This is the only new line added to this file (well and the closing brackets on the last line) everything else is just indented. The fix for the failing test here was fixed in the `fixtures.ts` above

Reply:
What's the benefit of this parent `describe` block?

Reply:
> Being able to run all of the test in the file all at once

@teodor2312 you can already run `yarn test src/commands/synthetics/__tests__/mobile.test.ts` to do exactly that 😁 

But it's true that we have one in other files. However, I think that it's superfluous to add the word "tests" in the name of the describe block, and we usually just use the file name:

```suggestion
describe('mobile', () => {
```

Reply:
I run tests from inside my VS Code though i and i have a little `Run | Debug` thing, which is what i need the big describe for, but yes also there's usually a big describe in other tests as well :D

<img width="343" alt="image" src="https://github.com/DataDog/datadog-ci/assets/25252536/8530e947-67bd-402a-98a3-357449e1e225">